### ISSUE TITLE
Skip link component

### ISSUE THREAD

**Issue by andredelramo-hm** [Telefonica/mistica-design#1968]
## Description
#1919 

Create a solution to avoid failing:
https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks.html

Based on the following technique
https://www.w3.org/WAI/WCAG22/Techniques/general/G1

Initial idea:

Create a component that appears only on keyboard tab that can be placed anywhere on the screen and allows to inset a list of links that will behave as anchors to navigate to another content in the same page.

Why it can be placed anywhere?

We cannot foresee how the content of a product is built and organized. It could be necessary no only to skin to the main content from the first tab link on the page, but to skip repeated pats of content as described in the book index example of the following technique:

https://www.w3.org/WAI/WCAG22/Techniques/general/G123

## Tasks

- [x] [Update specs](https://www.figma.com/design/xUVDR5AhW9m5uKNulAAfg6/%F0%9F%94%B8-%5BWIP%5D--Skip-link?m=auto&node-id=3232-4806&t=CjTTLg8ioGxWljkC-1)
- [x] Design documentation
- [x] Add to mobile library
- [x] Add to desktop library
- [ ] Add to UI Kit
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2144)
- [x] [UXW ticket](https://jira.tid.es/browse/PRODUCTDSN-3367)
- [x] [LOC ticket](https://jira.tid.es/browse/LOC-1888)
